packages/lesswrong/components/posts/RecombeePostsList.tsx

```diff
@@ -98,13 +101,28 @@ export const RecombeePostsList = ({ algorithm, settings, limit = 15, classes }:
   });
 
   const results: RecommendedPost[] | undefined = data?.[resolverName]?.results;
+
+  const recommId = uniq(results?.map(({recommId}) => recommId) ?? []).join('-');
```

Comment:
surely you want the recommIds to be inside of each item in `postIdsWithScenario`?  Or at least map the recommId to the scenario (so there's some way of associating them later)?